src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Grid,
+  Tab,
+  Typography,
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { TabContext, TabList, TabPanel } from '@material-ui/lab';
+import { useTranslation } from 'react-i18next';
+import { BasicTypes } from './index';
+import EditIcon from '@material-ui/icons/Edit';
+import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
+
+const useStyles = theme => ({
+  header: {
+    display: 'flex',
+    background: theme.palette.background.secondary,
+    color: '#FFFFFF',
+    marginLeft: '30px',
+    height: '50px',
+    paddingTop: '10px'
+  },
+  rightBar: {
+    textAlign: 'right',
+    display: 'flex',
+    alignItems: 'center',
+    margin: `0 ${theme.spacing(3)}px`
+  },
+  tabPanel: {
+    maxHeight: 300,
+    overflow: 'auto'
+  },
+  tabs: {
+    margin: '8px'
+  },
+  tab: {
+    minWidth: 0,
+    fontSize: '14px',
+    fontWeight: '500',
+    letterSpacing: '0',
+    lineHeight: '15px',
+    textAlign: 'center',
+    flexGrow: 1,
+    opacity: 1,
+    color: theme.palette.text.grey,
+    '&.Mui-selected': {
+      fontWeight: 'bold',
+      color: theme.palette.primary.contrastText
+    }
+  }
+});
+
+const ScenarioParameters = ({
+  classes,
+  editMode,
+  changeEditMode,
+  updateAndLaunchScenario,
+  workspaceId,
+  currentScenario,
+  scenarioId
+}) => {
+  // Translation
+  const { t } = useTranslation();
+
+  // General states
+  const [value, setValue] = useState('basic_types');
+  const [displayPopup, setDisplayPopup] = useState(false);
+
+  // States for parameters
+  const [textField, setTextField] = useState(currentScenario.data.parametersValues[0].value);
```

Comment:
Can we add a  comment to explain where the data comes from, and why the "0" index is hard-coded ?
(or a "TODO" comment to indicate that this part of code must be modified when the parameters types are changed)